DOCS-8625-WSC-index-kt (#926)
mulesoft/docs-connectors: web-service-consumer/1.6/modules/ROOT/pages/edits/index.adoc

= Web Service Consumer Connector 1.6 - Mule 4
:page-aliases: connectors::web-service/web-service-consumer.adoc

Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]

> Anypoint Connector for Web Service Consumer (Web Service Consumer Connector) consumes a SOAP web service from a Mule app, to acquire data from an external source. In many cases, you can use Anypoint Connector for Workday (Workday Connector) or Anypoint Connector for ServiceNow (ServiceNow Connector) to connect to a service provider, however; when no connector is available for a specific web service, the easiest way to consume the service is to use the Web Service Consumer Connector.

Reviewer: ```suggestion
Anypoint Connector for Web Service Consumer (Web Service Consumer Connector) consumes a SOAP web service from a Mule app to acquire data from an external source. In many cases, you can use Anypoint Connector for Workday (Workday Connector) or Anypoint Connector for ServiceNow (ServiceNow Connector) to connect to a service provider; however, when no connector is available for a specific web service, the easiest way to consume the service is to use the Web Service Consumer Connector.
```

Reviewer: Awesome topic intro!

> The main features of the connector include:

Reviewer: Because the items in this list are not parallel, it is difficult to find the right wording to introduces it.
This intro. refers to "features," so the list items should be nouns (features), not verbs.
But if we change the intro. to refer to user actions/goals, then we have the problem of the noun-based entries.
Can you think of a way to solve this?
My attempt is to make them all nouns, to go with "features," but I might have changed what you really intend to say, so please consider the following edits carefully and feel free to reword intro/list.

> * Consume DOC literal web services

Reviewer: ```suggestion
* Ability to consume DOC literal web services
```

> * Use SOAP multipart messages

Reviewer: ```suggestion
* Use of SOAP multipart messages
```

> * Use SOAP headers

Reviewer: ```suggestion
* Use of SOAP headers
```

> * Support DataSense for SOAP headers, SOAP body, and attachments

Reviewer: ```suggestion
* Support for DataSense for SOAP headers, SOAP body, and attachments
```

> * Use embedded DataWeave transformations inside the operation

Reviewer: ```suggestion
* Use of embedded DataWeave transformations inside the operation
```

> * Support an unified experience for SOAP with attachments and MTOM handling

Reviewer: ```suggestion
* Support for a unified experience for SOAP with attachments and MTOM handling
```

* Custom HTTP configuration as a transport

> * Support Web Service Security (WS Security)

Reviewer: ```suggestion
* Support for Web Service Security (WS Security)
```

[NOTE]

> Web Service Consumer Connector doesn't support RPC WSDLs.

Reviewer: Will your audience know what RPC WSDLs are?
If your novice users might not, consider spelling it out.

> For compatibility information and fixed issues, see the xref:release-notes::connector/connector-wsc.adoc[Web Service Consumer Release Notes]

Reviewer: Again, this is a new template change that Valkyrie and I discussed and approved:
```suggestion
For compatibility information and fixed issues, see Web Service Consumer Release Notes.
```
Move link to See Also.

== Prerequisites

To use this connector, you must be familiar with:

> * SOAP, WSDLs

Reviewer: ```suggestion
* SOAP and WSDLs
```

* Anypoint Connectors
* Mule runtime engine (Mule)
* Elements in a Mule flow
* Global elements
* DataWeave

> * Understand how to create a Mule app using Design Center or Anypoint Studio

Reviewer: parallelism
```suggestion
* Creating a Mule app using Design Center or Anypoint Studio
```

Before creating an app, you must have access to Anypoint Studio.

> //== Common Use Cases For the Connector

Reviewer: ```suggestion
//== Common Use Cases for the Connector
```

// Add a lead in sentence and then list common use cases for the connector

// For examples, see xref:<connector-name>-connector-examples.adoc[Examples].

== Audience

* New user
+
To create your Mule app, see:

> * xref:web-service-consumer-studio.adoc[Studio Configuration]

Reviewer: Should we add the reason why after the link, as do the following two?

* xref:web-service-consumer-config-topics.adoc[Additional Configuration Information]

> to help extend your understanding of the Web Service Consumer Connector

Reviewer: Is this really to extend knowledge of the connector or is it about customization?

//* xref:<connector-name>-connector-examples.adoc[Examples], which provides one or more use cases for the connector.
+
* Advanced user
+

> * Read the following:

Reviewer: Not a bullet
```suggestion
Read the following:
```

* xref:web-service-consumer-xml-maven.adoc[XML and Maven Support]
* xref:web-service-consumer-config-topics.adoc[Additional Configuration Information]

> //* xref:<connector-name>-connector-examples.adoc[Examples] topics

Reviewer: ```suggestion
//* xref:<connector-name>-connector-examples.adoc[Examples]
```
